test/utility-validation.js

```diff
@@ -28,17 +28,58 @@ describe('utility validation', function() {
         utilitySelectors: /\.[A-Z]+/,
         ignoreSelectors: /\.isok-[a-z]+/,
       };
+      var ignoreString = {
+        utilitySelectors: /\.[A-Z]+/,
+        ignoreSelectors: '\\.isok-[a-z]+',
+      };
+      var badIgnoreString = {
+        utilitySelectors: /\.[A-Z]+/,
+        ignoreSelectors: '(',
+      };
+      var emptyIgnoreString = {
+        utilitySelectors: /\.[A-Z]+/,
+        ignoreSelectors: '',
+      };
 
       it('accepts valid selectors', function(done) {
-        util.assertSuccess(done, '/** @define utilities */ .FOO {}', configWithIgnore);
+        var css = '/** @define utilities */ .FOO {}';
+
+        util.assertSuccess(function () {
+          util.assertSuccess(done, css, ignoreString);
+        }, css, configWithIgnore);
       });
 
       it('rejected invalid selectors that do not match ignore pattern', function(done) {
-        util.assertSingleFailure(done, '/** @define utilities */ .foo {}', configWithIgnore);
+        var css = '/** @define utilities */ .foo {}';
+
+        util.assertSingleFailure(function () {
+          util.assertSingleFailure(done, css, ignoreString);
+        }, css, configWithIgnore);
```

Comment:
Seems to me that these 3 lines should be reduced to one: `util.assertSingleFailure(done, css, ignoreString);`.

What's the reason you have it the way it is?